src/materialized/src/bin/materialized/main.rs

```diff
@@ -313,6 +317,26 @@ struct Args {
         value_name = "PATH"
     )]
     tls_key: Option<PathBuf>,
+    /// Specifies the tenant id when authenticating users. Must be a valid UUID.
+    #[clap(
+        long,
+        env = "FRONTEGG_TENANT",
+        requires_all = &["frontegg-jwk", "frontegg-api-token-url"],
+        hide = true
+    )]
+    frontegg_tenant: Option<String>,
+    /// JWK used to validate JWTs during user authentication as a PEM public
+    /// key. Can optionally be base64 encoded with the URL-safe alphabet.
+    #[clap(long, env = "FRONTEGG_JWK", requires = "frontegg-tenant", hide = true)]
+    frontegg_jwk: Option<String>,
+    /// The full URL (including path) to the api-token endpoint.
+    #[clap(
+        long,
+        env = "FRONTEGG_API_TOKEN_URL",
```

Comment:
since the env variables are named `FRONTEGG_TENANT`, `FRONTEGG_JWK` and `FRONTEGG_API_TOKEN_URL`, this won't interfere / attempt to get confused by the env variables we set in cloud currently (we set `MZCLOUD_JWK` and `MZCLOUD_TENANT`); this means we're pretty much fine to merge this.